tests/drivers/spi/spi_loopback/boards/nucleo_l552ze_q.overlay

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi1 {
+	pinctrl-0 = <&spi1_nss_pa4 &spi1_sck_pa5
+		     &spi1_miso_pa6 &spi1_mosi_pa7>;
+	dmas = <&dmamux1 0 12 0x440 0
+		&dmamux1 7 11 0x480 0>;
+	dma-names = "tx", "rx";
+	status = "okay";
+};
+
+&dma1 {
+	status = "okay";
+};
+
+&dma2 {
+	status = "okay";
+};
+
+&dmamux1 {
+	status = "okay";
+};
+
+&rcc {
```

Comment:
maybe  change of the rcc this is not really needed, could you please test wit and without,